inductiva/tasks/task.py

```diff
@@ -0,0 +1,147 @@
+"""Interact with a running/completed task on the Inductiva API."""
+import pathlib
+import time
+from typing import Dict, Any
+from typing_extensions import TypedDict
+
+from inductiva.client.models import TaskStatusCode
+from inductiva import api
+from inductiva.client.apis.tags.tasks_api import TasksApi
+from absl import logging
+
+
+class Task:
+    """Represents a running/completed task on the Inductiva API.
```

Comment:
Same.

Reply:
What do you suggest? This is just the one-liner, and I think reflects well what the class does.